juniper_official/Solutions/EX-Platform/PoE-KPI-Playbook.playbook

```diff
@@ -0,0 +1,19 @@
+/*
+ * Playbook contains 3 rules which monitors POE status , interface and controller 
+ * power consumption and notifies the status.
+ * 
+ * 1) Rule check-poe-total-power, checks POE Controller power consumption and raises 
+ *    alarm in case of anomalies.
```

Comment:
s/checks POE Controller power consumption and raises 
 *    alarm in case of anomalies/ checks POE Controller power consumption and raises 
 *    alarm in case of power consumption is above threshold/g